packages/plugin-amplitude-node/README.md

```diff
@@ -0,0 +1,3 @@
+# Amplitude Node Plugin
```

Comment:
For the plugins please add "for Iteratively SDK" at the end of the title and package.json descriptions. This are published as separate packages so it would be call out its our plugin vs Amplitude's etc. e.g. "Amplitude Node Plugin for Iteratively SDK"

Reply:
Added